forecasting/src/autogluon/forecasting/models/abstract/abstract_model.py

```diff
@@ -0,0 +1,3 @@
+class AbstractModel:
```

Comment:
may want to call this `AbstractForecastingModel` in the future to reduce confusion?